controllers/numaresourcesoperator_controller.go

```diff
@@ -299,6 +304,34 @@ func (r *NUMAResourcesOperatorReconciler) syncNUMAResourcesOperatorResources(ctx
 	return daemonSetsNName, nil
 }
 
+func (r *NUMAResourcesOperatorReconciler) deleteUnusedDaemonSets(ctx context.Context, instance *nropv1alpha1.NUMAResourcesOperator, mcps []*machineconfigv1.MachineConfigPool) error {
+
+	var daemonSetList appsv1.DaemonSetList
+	err := r.List(ctx, &daemonSetList, &client.ListOptions{Namespace: instance.Namespace})
+	if err != nil {
```

Comment:
nit: you can use one liner
```
if err := r.List(ctx, &daemonSetList, &client.ListOptions{Namespace: instance.Namespace}); err != nil {
    // do something
}

Reply:
Yeah I know but It seems a little bit harder to read to me than the two line expression ...